### PR TITLE
updated readme after removing minishift makefile targets in other repos

### PR DESCRIPTION
**Pull request by ranakan19**
fix wrt: https://issues.redhat.com/browse/CRT-904

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -14,20 +14,15 @@ This repository uses https://github.com/golang/go/wiki/Modules[Go modules]. You
 
 == Setting Up and Connecting Host and Member Clusters
 
-In a new terminal, execute the following commands
+In a new terminal, login as admin to host-operator and then execute the following commands
```
Comment (xcoulon) [inline README.adoc:17]:
```suggestion
In a new terminal, login as admin and execute the following commands from the local `host-operator` repository directory
```

File: README.adoc [hunk 2]
```diff
@@ -14,20 +14,15 @@ This repository uses https://github.com/golang/go/wiki/Modules[Go modules]. You
...
 ```
-$ cd $GOPATH/src/github.com/codeready-toolchain/host-operator
-$ minishift start --profile host
-$ make login-as-admin 
 $ make create-namespace 
 $ make deploy-rbac
 $ make deploy-crd
 ```
 
+login as admin to member and then execute the following commands
```
Comment (xcoulon) [inline README.adoc:24]:
```suggestion
then execute the following commands from the local `member-operator` repository directory
```

File: README.adoc [hunk 3]
```diff
@@ -183,13 +168,7 @@ make add-host-to-member
...
 
 ==== Using a single cluster for both operators host and member
-Make sure that you have started minishift. You can use following commands for it.
-
-* `minishift start`
-
-Once you have minishift or any openshift cluster running and logged in as admin, you can run `./scripts/add-clusters.sh` script with required flags.
-
-Assumption: This script assumes that you have host and member operators running on minishift or any other Openshift cluster
+Assumption: This script assumes that you have host and member operators running on Openshift cluster
```
Comment (xcoulon) [inline README.adoc:171]:
```suggestion
Assumption: This script assumes that you have both host and member operators running on the same OpenShift cluster
```

**Review by xcoulon — APPROVED** [review]

File: README.adoc [hunk 4]
```diff
@@ -14,20 +14,15 @@ This repository uses https://github.com/golang/go/wiki/Modules[Go modules]. You
 
 == Setting Up and Connecting Host and Member Clusters
```
Comment (MatousJobanek) [inline README.adoc:15]: From my point of view, this whole section is outdated, and we can drop it. If there is someone who needs to configure host & member operator, the (s)he should use https://github.com/codeready-toolchain/toolchain-e2e/blob/master/dev_install.adoc 
WDYT @xcoulon ?

File: README.adoc [hunk 5]
```diff
@@ -126,14 +119,6 @@ $ oc apply -f deploy/user.yaml
...
-==== host and member clusters using host and member profiles on minishift
-Make sure that you have started minishift as different profiles. You can use following commands for it.
-
-* `minishift start --profile host`
-* `minishift start --profile member`
-
-Once you have minishift running with above profiles, you can `./add-clusters.sh` script with required flags.
-
 Assumption: This script assumes that you have host and member operators running on host and member clusters(in our case clusters are different profiles)
 in respective namespace with all requires resources like Service Account, (Cluster)Role, (Cluste)RoleBinding, Custom Resource Definition.
```
Comment (MatousJobanek) [inline README.adoc:123]:
```suggestion
Assumption: This script assumes that you have host and member operators running on host and member clusters.
```

File: README.adoc (hunk 4 above)
Comment (MatousJobanek) [reply README.adoc:15]: A reminder :-)  You can remove the whole ` == Setting Up and Connecting Host and Member Clusters` part. Either now or in the next PR - as you wish :wink:

**Review by MatousJobanek — APPROVED** [review]

File: README.adoc (hunk 4 above)
Comment (ranakan19) [reply README.adoc:15]: And replace it with a note probably then?

**Pull request merged**